app/packages/components/src/components/Selection/Selection.tsx

```diff
@@ -79,10 +80,8 @@ function Selection(props: SelectionProps) {
     readonly,
     onEdit,
     onClear,
+    dataCy,
```

Comment:
I'd argue that this is a leaky abstraction. I'd instead take in some kind of `id` prop and use that as `data-cy` because technically a `Selection` component is not parametrized by `dataCy`.